src/garage/np/algos/base.py

```diff
@@ -25,4 +28,34 @@ def train(self, runner):
             The average return in last epoch cycle or None.
 
         """
-        pass
+
+    def evaluate_performance(self, batch):
+        # pylint: disable=no-self-use
+        r"""Evaluate the performance of the algorithm.
+
+        Args:
+            batch (dict): Evaluation trajectories, representing
```

Comment:
this is the wrong dtype. this interface should accept TrajectoryBatch (or the equivalent list, if you don't want to adopt it yet).